.travis.yml

```diff
@@ -27,3 +27,7 @@ script:
 
 after_success:
   - coveralls
+
+addons:
+  code_climate:
+    repo_token: 8b6b72db7cca0082ac23c0d86062b40eb3761d65bdfaafa85c491a2e0b95f8bd
```

Comment:
This token is ok to expose to the public.
But I'm actually gonna remove this thing since it's needed for coverage reporting to CodeClimate, which is not going to happen now.